lib/organisms/gravity-form/gravity-form.directive.js

```diff
@@ -2,8 +2,18 @@ angular
   .module('lnPatterns')
   .directive('lnOGravityForm', lnOGravityForm);
 
+/**
+ * TODO:
+ *    - Expand the list of supported input types (see comments in form parser)
+ *
+ *    - NOTE: The validation feedback / update mechanism isn't very 'Angular'
+ *      at the moment
+ *
+ *    - Migrate to use of ng-model for form data gathering
+ */
+
 /*@ngInject*/
```

Comment:
Ok, @adam wanted us to use @nginject though, so might be worth talking about that